extensions/hexpm/src/search-packages.tsx

```diff
@@ -0,0 +1,205 @@
+import {
+  ActionPanel,
+  Color,
+  Icon,
+  List,
+  showToast,
+  useNavigation,
+  Action,
+  Toast,
+  getPreferenceValues,
+} from "@raycast/api";
+import { useState } from "react";
+import { Package } from "./types";
+import { useDebounce } from "use-debounce";
```

Comment:
```suggestion
import { useDebounce } from "use-debounce";
```
Latest version of this package is a major breaking change without documentation.
I would love to see RayCast hook available in the `utils` package